apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -397,4 +453,8 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
       _ -> :error
     end
   end
+
+  defp function_exported_check?(module, function, arity) do
```

Comment:
nit: just call this `function_exported?` and add a comment indicating that it's here for testing.

Reply:
I initially thought the same, but this would conflict with Kernel.function_exported? and cause the compilation to fail. Do you have any other name suggestions?

Reply:
`exported?`